Linux/package/include/sgx_tsgxssl.edl

```diff
@@ -32,14 +32,18 @@
 /* sgx_tssl.edl - Top EDL file. */
 
 enclave {
-    
+   
     from "sgx_tstdc.edl" import *;
-    
+   
     untrusted {
         void u_sgxssl_ftime([out, size=timeb_len] void * timeptr, uint32_t timeb_len);
-        size_t u_sgxssl_write (int fd, [in,size=n]const void *buf, size_t n);
-        size_t u_sgxssl_read(int fd, [out, size=count]void *buf, size_t count);
+        ssize_t u_sgxssl_write (int fd, [in,size=n]const void *buf, size_t n);
+        ssize_t u_sgxssl_read(int fd, [out, size=count]void *buf, size_t count);
         int u_sgxssl_close(int fd);
+        int u_sgxssl_open([user_check]const char* fname, int flags);
```

Comment:
Not for this PR.
When these file related untrusted functions fails, `errno` represents error code.  `On error, -1 is returned, and errno is set appropriately.`
Can you add `propagate_errno` EDL key word to in order to pass `errno` to enclave?